lib/web/apiserver.go

```diff
@@ -3544,7 +3544,7 @@ func (h *Handler) siteSessionStreamGet(w http.ResponseWriter, r *http.Request, p
 
 	// look at 'offset' parameter
 	query := r.URL.Query()
-	offset, _ := strconv.Atoi(query.Get("offset"))
+	offset, err := strconv.Atoi(query.Get("offset"))
```

Comment:
This changes behavior. When skipping the error check we would return 0 and move on, now the error will be return to the caller (I'm assuming a few thing here). I think that we're fine with it?

Reply:
It's a fair point, but I believe it was the author's (Ev, hehe) original intention, as you can see we check `if err != nil` on line 3548.

Prior to this fix, we were just checking the wrong `err` from a previous call.

Given that this has been around for so long, I also have no problem with skipping the error check and defaulting to 0 (and removing the unnecessary and incorrect error check below). Come to think of it, that seems safest.